src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -270,6 +270,7 @@ public bool TryGetFlowAnalysisAnnotations(out FlowAnalysisAnnotations value)
                     if (inOutFlags == ParameterAttributes.Out)
                     {
                         refKind = RefKind.Out;
+                        scope = DeclarationScope.RefScoped;
```

Comment:
>scope = DeclarationScope.RefScoped; [](http://example.com/codeflow?start=24&length=35)

We allow to override it with an attribute?

Reply:
Yes, that's possible, with an explicit attribute from metadata.